app/services/planned_disbursement_history.rb

```diff
@@ -25,7 +25,10 @@ def set_value(value)
   end
 
   def clear
-    entries.destroy_all
+    entries.each do |entry|
+      entry.create_activity(key: "planned_disbursement.destroy", owner: @user, parameters: {associated_activity_id: @activity.id})
```

Comment:
👍  

Storing the `associated_activity_id` here seems sensible. I'm wondering if this warrants documenting (ADR or something else?) so that we use the same parameter when doing this sort of thing on other models.

Reply:
Yeah, I'm not sure exactly where to document this. An ADR seems like it'd be too much.